Controller/DbControllers/ArtistController.cs

```diff
@@ -25,11 +25,13 @@ protected ArtistController(IDatabaseContext context) : base(context, context.Art
         }
 
         // Can't this be a little bit more generic. Like update role or something??
-        public void MakeArtist(User user)
+        public void MakeArtist(int userId)
         {
+            var user = userController.GetItem(userId);
```

Comment:
Dit lijkt me overbodig?
Als je hieronder alleen maar de userId nodig hebt?

Reply:
We updaten user zelf ook, de role wordt geupdated in line 32 en 33

Reply:
Ik zag het al idd